components/database/index.php

```diff
@@ -50,7 +50,7 @@
     echo '<h2>First User using the query builder:</h2>';
     echo '<pre>';
 
-    $user = Capsule::table('users')->where('id', 1)->get();
+    $user = Capsule::table((new User)->getTable())->where((new User)->getKeyName(), 1)->get();
```

Comment:
Can you instead add this as a commented-out alternate suggestion? I like it but also like the simpler version so people can get how it works.